test/metabase/db/data_migrations_test.clj

```diff
@@ -189,3 +190,85 @@
         (testing "...nor should other groups that happen to exist"
           (is (= []
                  (perms group))))))))
+
+(deftest fix-click-through-test
+  (is (= {:id 1,
```

Comment:
A `(testing "... (#15014)"` blurb would be super-helpful for these tests. I can't count how many times I've grepped the code-base for `#<issue-number>` to find tests that fixed an issue similar to one I'm working on fixing